### PR TITLE
Add `AttachNetns` to wrap `bpf_program__attach_netns`

### PR DESCRIPTION
**Pull request by mozillazg**
https://github.com/aquasecurity/libbpfgo/issues/91

### CODE REVIEW

**Comment by geyslan** [comment]
This PR is following the same approach of https://github.com/aquasecurity/libbpfgo/pull/196. Since both `bpf_program__attach_netns()` and `bpf_program__attach_cgroup()` are calling `bpf_program__attach_fd()` just setting a specific target_name, this LGTM so far.

**Comment by geyslan** [comment]
@mozillazg could you provide a selftest to this? Thanks.

File: libbpfgo.go [hunk 1]
```diff
@@ -1426,6 +1427,33 @@ func (p *BPFProg) AttachKretprobe(kp string) (*BPFLink, error) {
...
+	if C.IS_ERR_OR_NULL(unsafe.Pointer(link)) {
+		return nil, errptrError(unsafe.Pointer(link), "failed to attach network namespace on %s to program %s", networkNamespacePath, p.name)
+	}
+
+	// fileName will be used in bpfLink.eventName. eventName follows a format
+	// convention and is used to better identify link types and what they are
+	// linked with in case of errors or similar needs. Having eventName as:
+	// netns-progName-/proc/self/ns/net would look weird so replace it
+	// to be netns-progName-proc-self-ns-net instead.
+	fileName := strings.ReplaceAll(networkNamespacePath, "/", "-")
```
Comment (geyslan) [inline libbpfgo.go:1446]: I think we need to skip index 0 of `networkNamespacePath`.

```suggestion
	fileName := strings.ReplaceAll(networkNamespacePath[1:], "/", "-")
```

**Comment by mozillazg** [comment]
> @mozillazg could you provide a selftest to this? Thanks.

@geyslan  Done.

image

**Review by rafaeldtinoco — APPROVED** [review]
LGTM. Thanks for the contribution!

**Pull request merged**